src/js/ui-events.js

```diff
@@ -3,11 +3,12 @@ import on from './on';
 import getAttribute from './get-attribute';
 import outer from './outer';
 
-export const EVENTS = Enum('click', 'keyup', 'enter', 'move', 'leave', 'Escape', 'Esc', 'touchstart', 'touchmove', 'touchend');
+export const EVENTS = Enum('click', 'change', 'keyup', 'enter', 'move', 'leave', 'Escape', 'Esc', 'touchstart', 'touchmove', 'touchend', 'input');
```

Comment:
Actually I never intended `ui-events` to be an accumulation of all possible events.
Since it's useful though I may would create a separate file 🤔

Reply:
:) make sense. I cant do it now thought for time reasons. Lets make an issue with all the things left

Reply:
fine with me